pkgs/applications/office/planner/default.nix

```diff
@@ -7,10 +7,10 @@
 , libtool
 , gnome2
 , libxslt
-, python
+, python2
 }:
 
-let version = "unstable-2018-03-25";
+let version = "unstable-2019-02-14";
```

Comment:
Why did you change the date? The commit was authored on Feb 13 (UTC)

https://gitlab.gnome.org/GNOME/planner/commit/76d31defae4979aa51dd37e8888f61e9a6a51367